### PR TITLE
create a containers directory for our Dockerfiles, add an initial example of a Dockerfile for Red

### PR DESCRIPTION
**Pull request by williamstark01**

### CODE REVIEW

**Comment by williamstark01** [comment]
@thibauthourlier creating a new repository just for Genebuild's (or even all Ensembl's) Dockerfiles is a good idea. Is there a plan to do that in the near future? If we don't go ahead with that soon, I think this PR should be merged instead of staying in limbo.

File: containers/Red/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+build a Docker image
+```
+docker image build -t williamebi/red:2.0 .
+```
+
+run the Docker image locally
+```
+# print the help message
+docker run williamebi/red:2.0
+
```
Comment (thibauthourlier) [inline containers/Red/README.md:25]: I think this is something we need to make sure that every one creating an image need to have the help printed when running without argument

**Review by thibauthourlier — APPROVED** [review]
I am not sure of the reasons for having the "make" image and the binary image. Maybe the make image should have the possibility of passing the CPU architecture if someone wants to use all the features from their CPU

File: containers/Red/README.md (hunk 1 above)
Comment (williamstark01) [reply containers/Red/README.md:25]: Yes, it would be really helpful.

**Comment by williamstark01** [comment]
> I am not sure of the reasons for having the "make" image and the binary image. Maybe the make image should have the possibility of passing the CPU architecture if someone wants to use all the features from their CPU

I'm adding some more context about this in the README.

**Pull request merged**